src/utils/hendelseFilteringUtils.tsx

```diff
@@ -92,27 +92,16 @@ export const filterOnPersonregister = (
   }
 
   const filtered = Object.entries(personregister).filter(([, personData]) => {
-    if (filter.onskerMote && personData.harMotebehovUbehandlet) {
-      return true;
-    } else if (
-      filter.arbeidsgiverOnskerMote &&
-      personData.harOppfolgingsplanLPSBistandUbehandlet
-    ) {
-      return true;
-    } else if (filter.dialogmotekandidat && personData.dialogmotekandidat) {
-      return true;
-    } else if (filter.ufordeltBruker && !personData.tildeltVeilederIdent) {
-      return true;
-    } else if (filter.dialogmotesvar && personData.harDialogmotesvar) {
-      return true;
-    } else if (filter.aktivitetskrav && personData.aktivitetskravActive) {
-      return true;
-    } else if (
-      filter.behandlerdialog &&
-      personData.harBehandlerdialogUbehandlet
-    ) {
-      return true;
-    }
+    return (
+      (!filter.onskerMote || personData.harMotebehovUbehandlet) &&
+      (!filter.arbeidsgiverOnskerMote ||
+        personData.harOppfolgingsplanLPSBistandUbehandlet) &&
+      (!filter.dialogmotekandidat || personData.dialogmotekandidat) &&
+      (!filter.ufordeltBruker || !personData.tildeltVeilederIdent) &&
+      (!filter.dialogmotesvar || personData.harDialogmotesvar) &&
+      (!filter.aktivitetskrav || personData.aktivitetskravActive) &&
+      (!filter.behandlerdialog || personData.harBehandlerdialogUbehandlet)
+    );
```

Comment:
Mulig jeg ikke helt skjønner dette, men for meg ser det riktigere ut med: 
```
(filter.onskerMote && personData.harMotebehovUbehandlet) ||
(filter.arbeidsgiverOnskerMote && personData.harOppfolgingsplanLPSBistandUbehandlet) ||
(filter.dialogmotekandidat && personData.dialogmotekandidat) ||
osv
```
I tillegg kunne man kanskje at en boolean på filter som heter used, sånn at man kan ha 

`|| !filter.used`

til slutt. Dvs used må da returnere true hvis minst ett av filter-flaggene er true.

Reply:
Det du foreslår er vel det samme som vi hadde tidligere? (Bare i én if i stedet for flere if/else) 🤔 Da ender vi jo opp med en liste av folk som har én eller flere av de valgte filtrene; men nå skal vi ha det motsatte, altså kun de som matcher alle valgte filtre.